etc/kayobe/kolla/config/prometheus/rabbitmq.rules

```diff
@@ -6,7 +6,7 @@ groups:
 - name: rabbitmq.rules
   rules:
   - alert: RabbitMQNodeDown
-    expr: sum(rabbitmq_build_info{instance!=""}) < 3
+    expr: sum(rabbitmq_build_info{instance!=""}) < {% endraw %}{{ groups['rabbitmq'] | length }}{% raw %}
```

Comment:
Sorry as I hit send I realised the raw tags are different here.

``rabbitmq`` is a group defined in Kolla-Ansible, rather than Kayobe, so it think we want this to be included in the whole file's raw tags. 
```suggestion
    expr: sum(rabbitmq_build_info{instance!=""}) < {{ groups['rabbitmq'] | length }}
```